### DISCUSSION TITLE
Revert UI Changes (# of Images, Fix Faces Checkbox)

### DISCUSSION THREAD

**Discussion by Maergoth** [hlky/stable-diffusion-webui#448] [Q&A] [answered]
I just reinstalled to try to get some new features working. In doing so, I'm dealing with major UI changes. I have both GFGAN and ESRGAN installed properly (as far as I can tell) but the only "Face Fix" option is in the image lab, and there seems to be no easy way to port outputs INTO the image lab directly. Also the "Upscale W/ESRGAN" button seems to not function.

Not sure if part of the issue is this:
![image](https://user-images.githubusercontent.com/93102337/187969166-0639cd46-6271-4f91-9b57-e9df357c60cf.png)
This happens every time I run it.

Is there any way to specifically rollback to some UI Elements from a previous version?

Specifically:

Number of Images to Generate (Up to 250)
"Fix Faces" checkbox directly in the Text-to-Image Interface

![image](https://user-images.githubusercontent.com/93102337/187968189-565c19e1-b0e6-4f07-b209-749783144209.png)
This is what my GUI looks like. Not sure if I'm supposed to be seeing more than this, or if the steps to install GFGAN and ESRGAN have changed? 

![image](https://user-images.githubusercontent.com/93102337/187968301-5b1a587c-6375-45fc-9cdc-0df8e113fca7.png)

**Comment by altryne** [comment, marked as answer]
No rolling back, number of images was down to sensible defaults that will make this work for most users. 

You can override that number with a dynamic param as outlined in here : 
#421  

Upscalers have moved to the Image lab tab, as we plan to keep adding more, it makes sense to unify them

**Reply by Maergoth** [reply]
That makes sense, but there's no way to fix faces in the same workflow as generating the text-to-images? There used to be a checkbox so you didn't need to take a second step for each generated image

**Comment by karmicfix** [comment]
UI changes are good. I amanged to get the numbe rof images to be generated higher. Now the onyl pending issue is, that earlier there was a waifu diffusion shortcut which directyl launched fast without any pip dependency installs. Why does it have to run te pip dependencies everytime, si there any way to avoid that part. This tkes much more time to launch now. Please advice.

**Reply by karmicfix** [reply]
![image](https://user-images.githubusercontent.com/112202619/188005470-e17677e9-bd05-413c-aa97-872e13ce9c13.png)
